octofree/fix_octopus_spacing.py

```
@@ -0,0 +1,125 @@
+        frame_num: Frame number (0-3) for animation cycle
+    
+    Returns:
+        List of strings representing the octopus art
+    \"\"\"
+    frames = [
+        # Frame 0 - neutral
+        [""")
+    
+    for frame_num, frame in enumerate(fixed_frames):
+        if frame_num > 0:
+            print(f"        ],")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)